architecture/README.md

```diff
@@ -0,0 +1,3 @@
+### Relevant articles
+
+- [A Quick and Practical Example of Hexagonal Architecture in Java](https://www.baeldung.com/a-quick-and-practical-example-of-hexagonal-architecture-in-java-3/)
```

Comment:
remove, we do this in the end after the article is published

Reply:
Removed in new PR. We should update _baeldung - for authors - 4 - implementation code_ to say that the README.md should be updated after the article is published. Should I update the README.md file in the hibernate types sub-module.